lib/cdo/graphics/certificate_image.rb

```diff
@@ -32,6 +21,16 @@ def create_certificate_image2(image_path, name, params={})
 
 # applies the given text to the given image object.
 def apply_text(image, text, pointsize, font, color, x_offset, y_offset)
+  # If there is no text, don't try to render it.
```

Comment:
I'm not super familiar with this code path, are you changing the behavior from "rendering a certificate if name is empty" to no longer render it?

Reply:
How rendering work is you create an image which just has text in it, then you merge it with the existing certificate image. This changes it so we don't even attempt to create that temporary "text image" because it will just be empty (and ImageMagick will throw an exception).

Reply:
Got it.